pitch.md

```diff
@@ -1,44 +1,160 @@
-# Project Title (replace me)
+# ComplexCalendar
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about making complex calendar's easy to make. For any calendar
+software it is easy to make specific events but for most people, their life
+heavily revolves around a core calendar that repeats and doesn't change often.
+I want to make it easier for users to define their complex calendars in fewer
+words. 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The user for this project would be someone that has a highly structured calendar
+that is too complex to easily define with a simple repeat item button. An example
+of one user would be a college student. Their calendar for classes only applies 
+when school is in session. It would be much easier for the student to define a 
+school section of their calendar and mark the dates of the school year to apply
+it. This would eliminate their need to go months forward in the calendar to set 
+up new repeated events that were already set up for other times. 
```

Comment:
This seems like something that should already exist, which I think makes it a really good choice.  Like Prof. Ben said at the beginning of the semester, sometimes you'll find something and think 'how come we don't always do it this way' and I think this might be an example of that.